test/scripts/prc_256.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
what about doing this script in pure r2 instead of depending on python?

also im not sure why are we accepting flags with numbers as name, this seems like regression to me. the flagnames should be foo.%02x. also why are the flags needed when px doesnt needs them?

Reply:
> what about doing this script in pure r2 instead of depending on python?

oh ok 6c95e7a4e95dd2b496f63dc25d573b2c34531ceb ... i think `@@s` has an off-by-one error

> also why are the flags needed when px doesnt needs them?

the flags are needed for the test itself i.e. for its fancy color table ... the hexpairs below are actually flag names:

![prc_color_table](https://user-images.githubusercontent.com/12002672/82151300-7ccf6a00-988d-11ea-9b78-a51eed8a4a2e.png)

> also im not sure why are we accepting flags with numbers as name, this seems like regression to me.

emm why not? ... anyway currently `prc` doesn't cut off any prefixes when displaying flag names ... probably it should, but that's a topic for another pr